infrastructure/variables.tf

```diff
@@ -3,5 +3,5 @@ variable "product" {}
 variable "env" {}
 
 variable "common_tags" {
-    type = map(string)
+  type = map(string)
 }
```

Comment:
a new line?

Reply:
Autogenerated file added this, don't think this needs changing?